EIPS/eip-5757.md

```diff
@@ -0,0 +1,95 @@
+status: Draft
+type: Meta
+created: 2022-09-30
+requires: 1
+---
+
+## Abstract
+
+Ethereum improvement proposals (EIPs) occasionally link to resources external to this repository. This document sets out the requirements for origins that may be linked to, and the process for approving a new origin.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+### Definitions
+
+ - **Link**: Any method of referring to a resource, including: markdown links, anchor tags (`<a>`), images, citations of books/journals, and any other method of referencing content not in the current resource.
+ - **Resource**: A web page, document, article, file, book, or other media that contains content.
+ - **Origin**: A publisher/chronicler of resources, like a standards body (eg. w3c) or a system of referring to documents (eg. Digital Object Identifier System).
+
+### Requirements for Origins
+
+Permissible origins **SHOULD NOT** obviously allow redistributing their resources as part of the EIP repository. If an origin does obviously allow redistribution of resources, include those resources in an appropriate directory within this repository and link to those copies instead. The common exception to this rule are git repositories under the Ethereum organization.
+
+Permissible origins **MUST** provide a method of uniquely identifying a particular revision of a resource. Examples of such methods may include git commit hashes, version numbers, or publication dates.
+
+Permissible origins **MUST** have a proven history of availability. A origin existing for at least ten years and reliably serving resources would be sufficient—but not necessary—to satisfy this requirement.
+
+Permissible origins **MUST NOT** charge a fee for accessing resources.
+
+### Origin Removal
+
+Any approved origin that ceases to satisfy the above requirements **MUST** be removed from [EIP-1](./eip-1.md). If a removed origin later satisfies the requirements again, it MAY be re-approved by creating a new EIP as described in [Origin Approval](#origin-approval).
+
+Finalized EIPs (eg. those in the `Final` or `Withdrawn` statuses) **SHOULD NOT** be updated to remove links to these origins.
+
+Non-Finalized EIPs **MUST** remove links to these origins before changing statuses.
+
+### Origin Approval
+
+#### Proposal
+
+Each potential permitted origin **SHALL** be described in its own EIP.
+
+These EIPs **MUST** describe how the origin meets the requirements described above.
+
+These EIPs **MUST** include the markup for creating a link that satisfies all the above requirements and an example of such a link.
+
+When opened, these EIPs:
+
+ - **MUST** have a `status` of `Draft`.
+
+Throughout their lifetime, these EIPs:
+
+ - **MUST** have a `type` of `Meta`;
```

Comment:
How about we add a `category` for `Meta` EIPs?

Reply:
That would certainly be better than a special title prefix.